packages/discovery-provider/src/api/v1/users.py

```diff
@@ -1071,17 +1071,15 @@ class TrackHistoryFull(Resource):
     def _get(self, id, authed_user_id):
         args = track_history_parser.parse_args()
         decoded_id = decode_with_abort(id, ns)
-        current_user_id = get_current_user_id(args)
```

Comment:
Oops note to self (and maybe question for @rickyrombo):
Since we're no longer consuming this value, the SDK probably shouldn't indicate that you can send it?

Reply:
Update: Figured it out: 719d78e27badbb2dcc914d032566959efe1069c4